homeassistant/components/gpsd/sensor.py

```diff
@@ -120,9 +121,9 @@ def __init__(
     def native_value(self) -> str | None:
         """Return the state of GPSD."""
         if self.agps_thread.data_stream.mode == 3:
-            return "3D Fix"
+            return "3d_fix"
         if self.agps_thread.data_stream.mode == 2:
-            return "2D Fix"
+            return "2d_fix"
```

Comment:
This is a breaking change

Reply:
I'll revert it, I don't think there is a valid point to translate that anyways.

Reply:
I mean, I don't mind a breaking change, you only have to write a small thing that it's now breaking and people should update automations

Reply:
Description updated. Thanks.